gaphor/ui/greeter.glade

```diff
@@ -0,0 +1,191 @@
+                        <property name="spacing">6</property>
+                        <child>
+                          <object class="GtkLabel">
+                            <property name="visible">True</property>
+                            <property name="can-focus">False</property>
+                            <property name="label" translatable="yes">Recent files</property>
+                            <property name="xalign">0</property>
+                            <attributes>
+                              <attribute name="weight" value="bold"/>
+                            </attributes>
+                            <style>
+                              <class name="dim-label"/>
+                            </style>
+                          </object>
+                          <packing>
+                            <property name="expand">False</property>
+                            <property name="fill">True</property>
+                            <property name="position">0</property>
+                          </packing>
+                        </child>
+                        <child>
+                          <object class="GtkFrame">
+                            <property name="visible">True</property>
+                            <property name="can-focus">False</property>
+                            <property name="label-xalign">0</property>
+                            <property name="shadow-type">in</property>
+                            <child>
+                              <object class="GtkListBox" id="greeter-recent-files">
+                                <property name="visible">True</property>
+                                <property name="can-focus">False</property>
+                              </object>
+                            </child>
+                          </object>
+                          <packing>
+                            <property name="expand">False</property>
+                            <property name="fill">True</property>
+                            <property name="position">1</property>
+                          </packing>
+                        </child>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+              </object>
+              <packing>
+                <property name="name">recent-files</property>
+              </packing>
+            </child>
+            <!-- new stack -->
+            <child>
+      <object class="GtkLabel">
+        <property name="visible">True</property>
+        <property name="can-focus">False</property>
+        <property name="label" translatable="yes">&lt;big&gt;&lt;b&gt;Welcome to Gaphor&lt;/b&gt;&lt;/big&gt;
+
+It's time to create your first model
+or open an existing model.
+
+Happy modeling!
+
+• &lt;a href="https://gaphor.org"&gt;Gaphor homepage&lt;/a&gt;
+• &lt;a href="https://github.com/gaphor"&gt;GitHub&lt;/a&gt;
```

Comment:
Did you mean to link this to the organization instead of the repo?

Reply:
Yes, that was intentional :)